src/udf/python/setup.py

```diff
@@ -5,17 +5,17 @@
 
 setup(
     name="risingwave",
-    version="0.0.4",
+    version="0.0.5",
```

Comment:
Why version is updated?

Reply:
It should have been updated in an earlier PR, but I forgot it...